inventory

```diff
@@ -16,6 +16,9 @@ template_service_broker_install=false
 openshift_use_manageiq=false
 openshift_install_examples=false
 
+# Docker
+container_runtime_docker_storage_type=overlay2
```

Comment:
afaik is overlay2 is not recommended for production.
I think we should stay on production side of configuration, no?

Reply:
I think that you refer to `devicemapper` with `loopback-lvm`